app/Models/Auth.ts

```diff
@@ -0,0 +1,51 @@
+import { DateTime } from 'luxon'
+import Hash from '@ioc:Adonis/Core/Hash'
+import { column, beforeSave, BaseModel } from '@ioc:Adonis/Lucid/Orm'
+import { v4 as uuidv4 } from 'uuid'
+
+export default class User extends BaseModel {
+  @column({ isPrimary: true })
+  public id: string
+
+  @column({ meta: { unique: true } })
+  public email: string
+
+  @column({ serializeAs: null })
+  public password: string
+
+  @column({ meta: { type: 'enum', enum: ['male', 'female'] } })
+  public gender: string
+
+  @column()
+  name: string
+
+  @column({ meta: { unique: true } })
+  mobile: string
+
+  @column()
+  dob: DateTime
```

Comment:
All the extra information of the user should be in a new table called "UserProfile".

Please go through the task that I've given you once again.